cr-ideas.md

```diff
@@ -0,0 +1,8 @@
+# Corbin's Random Idea's
+
+- Budget Web App using Plaid's API
+- Personal Asset Management System (copyright, Corbin Roth 2022)
+    - Think of a personal form builder centered around the things you own (assets). For example, what year is your vehicle? VIN#? Wheel Size? These are all pieces of info we need...but don't need all the time. I need my cars wheel size when replacing a tire... I need my vent size when ordering new filters from Amazon etc.
+
+- Specific Nashville Traffic Map / makes better routing decisions based on Nashville norms.
+    - IDK How we would do this...there is an awesome API/SDK out there but I am pretty sure it costs $$$.
```

Comment:
Very interesting. We would have to compare live traffic data with the hidden streets of Nashville that Waze wouldn't take you down. Sounds like a fun challenge!